include/class.ticket.php

```diff
@@ -235,14 +235,8 @@ function getPid() {
         return $this->ticket_pid;
     }
 
-    function getChildTickets($pid) {
-        return Ticket::objects()
-                ->filter(array('ticket_pid'=>$pid))
-                ->values_flat('ticket_id', 'number', 'ticket_pid', 'sort', 'thread__id', 'user_id', 'cdata__subject', 'user__name', 'flags')
-                ->annotate(array('tasks' => SqlAggregate::COUNT('tasks__id', true),
-                                 'collaborators' => SqlAggregate::COUNT('thread__collaborators__id'),
-                                 'entries' => SqlAggregate::COUNT('thread__entries__id'),))
-                ->order_by('sort');
+    function getChildren() {
+        return self::getChildTickets($this->getId());
```

Comment:
I would assume it would be a good idea to cache the resultset here.

```php 
function getChildren() {
    if (!isset($this->_children))
        $this->_children = self::getChildTickets($this->getId());

    return $this->_children;
}
```